sklearn/preprocessing/data.py

```diff
@@ -1901,3 +1950,526 @@ def transform(self, X):
+    distribution. Note that this transform is non-linear. It may distort linear
+    correlations between variables measured at the same scale but renders
+    variables measured at different scales more directly comparable.
+
+     Read more in the :ref:`User Guide <preprocessing_transformer>`.
+
+    Parameters
+    ----------
+    n_quantiles : int, optional (default=1000)
+        Number of quantiles to be computed. It corresponds to the number
+        of landmarks used to discretize the cumulative density function.
+
+    output_distribution : str, optional (default='uniform')
+        Marginal distribution for the transformed data. The choices are
+        'uniform' (default) or 'normal'.
+
+    ignore_implicit_zeros : bool, optional (default=False)
+        Only applies to sparse matrices. If True, the sparse entries of the
+        matrix are discarded to compute the quantile statistics. If false,
+        these entries are treated as zeros.
+
+    subsample : int, optional (default=1e5)
+        Maximum number of samples used to estimate the quantiles for
+        computational efficiency.
+
+    smoothing_noise : float, optional
+        Perturbs features at training time before computing quantiles by adding
+        Gaussian noise with standard deviation ``smoothing_noise``. It eases
+        the interpratation of the computed ``quantiles_`` when a particular
+        feature value is predominant.
+
+    random_state : int, RandomState instance or None, optional (default=None)
+        If int, random_state is the seed used by the random number generator;
+        If RandomState instance, random_state is the random number generator;
+        If None, the random number generator is the RandomState instance used
+        by np.random. Note that this is used by subsampling and smoothing
+        noise.
+
+    copy : boolean, optional, (default=True)
+        Set to False to perform inplace scaling and avoid a copy (if the input
+        is already a numpy array).
+
+    Attributes
+    ----------
+    quantiles_ : ndarray, shape (n_quantiles, n_features)
+        The values corresponding the quantiles of reference.
+
+    references_ : ndarray, shape(n_quantiles, )
+        Quantiles of references.
+
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from sklearn.preprocessing import QuantileTransformer
+    >>> RNG = np.random.RandomState(0)
+    >>> X = np.sort(RNG.normal(loc=0.5, scale=0.25, size=(25, 1)), axis=0)
+    >>> qt = QuantileTransformer(n_quantiles=10, random_state=0)
+    >>> qt.fit_transform(X) # doctest: +ELLIPSIS
+    array([...])
+
+    See also
+    --------
+    quantile_transform : Equivalent function without the object oriented API.
+
+    StandardScaler : perform standardization that is faster, but less robust
+    to outliers.
+
+    RobustScaler : perform robust standardization that removes the influence
+    of outliers but does not put outliers and inliers on the same scale.
```

Comment:
The formating of this section is broken:

https://9882-843222-gh.circle-artifacts.com/0/home/ubuntu/scikit-learn/doc/_build/html/stable/modules/generated/sklearn.preprocessing.QuantileTransformer.html#sklearn.preprocessing.QuantileTransformer

Please check the "See also" section of those classes to find examples or correct formatting.